src/Symfony/Component/Validator/Tests/ConstraintTest.php

```diff
@@ -269,4 +270,11 @@ public function testSetTypedProperty()
 
         $this->assertSame('bar', $constraint->foo);
     }
+
+    public function testSetDefaultPropertyChoice()
```

Comment:
I think it should rather be in a `ChoiceTest` as it is specific to this constraint. I know it doesn't exist yet. Let's create it.

Reply:
We have such tests for other contraints already, so yes, please create a dedicated `Choice Test`.

Reply:
👍